.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+- '3.6'
+
+services:
+  - docker
+
+before_install:
+  - docker build -t datawig-sagemaker .
+  - docker images
+#  - docker run
+#  - docker ps -a
+
+script:
+  - ./test/train_local.sh datawig-sagemaker
+  - ./test/serve_local.sh datawig-sagemaker & sleep 10 && ./test/http_request.sh
```

Comment:
Not a top priority but I'd rather remove the sleep here, and add resiliency (with retry and sleep's) into `./test/http_request.sh`.